rust/crd/Cargo.toml

```diff
@@ -13,7 +13,7 @@ serde = "1.0"
 serde_json = "1.0"
 snafu = "0.7"
 rand = "0.8"
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.13.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.14.0" }
```

Comment:
Maybe we could upgrade to 0.15 while we're at it? Not *needed* though

Reply:
done

Reply:
nice